tasks/skylab/FeatureCounts.wdl

```diff
@@ -0,0 +1,61 @@
+version 1.0
+
+task CountAlignments {
+
+  input {
+    File input_bam
+    File annotation_gtf
+
+    #runtime values
+    String docker = "quay.io/humancellatlas/snss2-featurecount:0.1.0"
```

Comment:
I thought at standup @khajoue2 said the version of featurecount we were using had a 2 in it, like 2.0.1 ? Or is the version number used for the docker name different?

Reply:
This is the docker version. The Dockerfile has the correct version.